File: CS/TimestampWebService/TimestampService.asmx.cs
using System;
using System.Data;
using System.Web;
using System.Collections;
using System.Web.Services;
using System.Web.Services.Protocols;
using System.ComponentModel;

namespace TimestampWebService {
	[WebService(Namespace = "http://localhost/")]
	[WebServiceBinding(ConformsTo = WsiProfiles.BasicProfile1_1)]
	[ToolboxItem(false)]
	public class TimestampService : System.Web.Services.WebService {
		[WebMethod]
		public DateTime GetTime() {
			return DateTime.Now;
		}
	}
}

File: VB/TimestampWebService/TimestampService.asmx.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Data
Imports System.Web
Imports System.Collections
Imports System.Web.Services
Imports System.Web.Services.Protocols
Imports System.ComponentModel

Namespace TimestampWebService
	<WebService(Namespace := "http://localhost/"), WebServiceBinding(ConformsTo := WsiProfiles.BasicProfile1_1), ToolboxItem(False)> _
	Public Class TimestampService
		Inherits System.Web.Services.WebService
		<WebMethod> _
		Public Function GetTime() As DateTime
			Return DateTime.Now
		End Function
	End Class
End Namespace
